docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -680,6 +680,18 @@ quarkus.kubernetes.readiness-probe.period=45s
 In this example `initial-delay` and `period` are fields of the type `Probe`.
 Below you will find tables describing all available types.
 
+==== Client Connection Configuration
+The underline Kubernetes client connection can be configured through properties defined by the Kubernetes Client runtime.
+These underline client connection gets automatically configured according to your local connection (by `kubectl`).
+
+In certain situations, you may need to tweak these properties like when your cluster API endpoint uses a self-signed SSL Certificate. In this case, you need to explicitly configure the client to trust your cluster API certificate. You can achieve this by defining the following property:
+
+[source,properties]
+----
+quarkus.kubernetes-client.trust-certs=true
+----
+
+Note that the full list of Kubernetes client properties is available in the https://github.com/quarkusio/quarkus/blob/main/extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientBuildConfig.java[KubernetesClientBuildConfig] class.
```

Comment:
We can use the generated documentation reference 
```suggestion
{generated-dir}/config/quarkus-kubernetes-client.adoc
```

Reply:
See https://github.com/quarkusio/quarkus/blob/7a4201a057c3c52ac428db19b14594ee9824fe61/docs/src/main/asciidoc/datasource.adoc#reactive-datasource-configuration-reference for example on how to include it